thorlcr/msort/tsortmp.cpp

```diff
@@ -77,7 +77,7 @@ bool SortSlaveMP::sendRecv(CMessageBuffer &mb, unsigned timeout)
             mb.read(err);
             StringAttr errstr;
             mb.read(errstr);
-            throw MakeStringException(err, "%s", errstr.get());
+            throw MakeStringException(err, "SortSlaveMP(error from slave %u) - %s", (unsigned)rank, errstr.get());
```

Comment:
That may be more meaningful to you, but is it really much more meaningful to anyone else that might see it? What does SortSlaveMP mean ?